index.html

```diff
@@ -1651,10 +1651,11 @@ <h3>Credential Subject</h3>
           <dt><dfn class="export"
                    id="defn-credentialSubject">credentialSubject</dfn></dt>
           <dd>
-The value of the <code>credentialSubject</code> <a>property</a> is defined as
-a set of objects that MUST contain one or more claims that are each related to a
-<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
-<code>id</code>, as described in Section <a href="#identifiers"></a>.
+The value of the <code>credentialSubject</code> <a>property</a> is defined as a
+set of objects where each object MUST contain one or more <a>claims</a>
+expressed as property-value pairs of the <a>subject</a>. Each object MAY
```

Comment:
"the subject" seems ambiguous here between just being a syntactic part of a claim, and asking verifiers to check which objects are the credential's subjects. If you're focusing on the claim aspect, how about:

```suggestion
set of objects where each object MUST be the <a>subject</a> of one or more <a>claims</a>,
which MUST be serialized inside the <code>credentialSubject</code> <a>claim</a>.
Each object MAY
```

I'm not sure about the "which MUST be serialized" bit, but it seems like you do intend to forbid a serialization with `credentialSubject: "some:url"` where `"some:url"` gets its claims from somewhere else in the credential.

Reply:
That's better, thank you.